### DISCUSSION TITLE
Make numeric entries that haven't reached target gray

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#1117] [Feature Requests]
For numerical habits, in the sections History and Calendar, would you please consider making the entries that haven't met the set target gray? 
For example, if my target is 8k for a yellow coloured habit, only the entries 8000 or more should be shown in yellow, entries 7999 or less should be grayed.
That would be very helpful and it would make the mentioned sections more informative, especially Calendar.

![Screenshot_2021-09-23-09-52-36-020_org isoron uhabits](https://user-images.githubusercontent.com/34518046/134480817-d23ac8db-7c32-478f-9dd4-eb03b67dcaed.jpg)

**Comment by LordJon** [comment]
I agree that this would make more sense. This is probably also related to: #1058 #949 #939
